### ISSUE TITLE
Produce Windows build without elevate.exe

### ISSUE THREAD

**Issue by davidjgraph** [jgraph/drawio-desktop#47]
The Windows build contains a user elevation option in, by default. It's main use is in the updater, which we don't use. Investigate whether we should turn it off, or produce 2 builds for Windows.

https://github.com/MariaDima/electron-builder/blob/037ceaabb76c1324c34de9a63a9be4bd407517d6/docs/Options.md

Note that elevate can only be invoked if the user installs specifically as admin.

**Comment by hmalphettes** [comment]
Thanks @davidjgraph for this.

At the moment, the portable build is not on the 9.3.1 release page.

Here is the appveyor's build: https://ci.appveyor.com/project/davidjgraph/drawio-desktop/builds/19896070

It reports that the portable was generated but did not upload it to github:

```
  • uploading       file=draw.io-9.3.1-portable.exe provider=GitHub
  • GitHub release not created reason=existing type not compatible with publishing type tag=v9.3.1 version=9.3.1 existingType=release publishingType=draft
  • skipped publishing file=draw.io-9.3.1-portable.exe reason=existing type not compatible with publishing type tag=v9.3.1 version=9.3.1 existingType=release publishingType=draft
```

This seem to say that the release was already tagged and appveyor refused to add to it.

As a fallback I searched for artifacts attached to the appveyor job but that list is empty.

Is this something I can look into or simply wait for the next tag and it will be there?

**Comment by davidjgraph** [comment]
Yeah, sorry, that workflow wasn't clear. I've pulled the change, there just hasn't been a release since.

There were a couple of severe fixes I needed out, so I released with those and started looking at the other PRs. Problem is that pulling closes the PR and that marks it as done on board.

It shouldn't be too long before a release, it's just a manual process to sign the windows release using a key on a physical dongle and I'm not a Windows user so I have to fire up a machine in the back of a cupboard... 😎
